pkg/harvester/components/novnc/NovncConsoleWrapper.vue

```diff
@@ -257,6 +257,10 @@ export default {
       });
     },
 
+    reconnect() {
+      window.location.reload();
+    },
```

Comment:
Do we need to clarify the requirement? To me, reconnect and reload are different:
- Reconnect: If the 10 automatic retries are not enough, we could keep trying (e.g., 100 times)
- Reload: If even 100 retries cannot establish a connection, then we may need to reload the page